packages/rdbg/rdbg.1.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "1.2"
+maintainer: "erwan.jahier@imag.fr"
+bug-reports: "erwan.jahier@imag.fr"
+authors: [ "Erwan Jahier" ]
+license: "GPL-3"
+homepage: "http://rdbg.forge.imag.fr/"
+dev-repo: "https://forge.imag.fr/anonscm/git/rdbg/rdbg.git"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocamlfind" "remove" "rdbg-plugin"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "base-unix"
+  ("camlp4" {= "4.03.0"} | "camlp4" {>= "4.02.1+system"} | "camlp4" {= "4.02.1+2"} | "camlp4" {= "4.02.1+1"} | "camlp4" {= "4.02.1"} | "camlp4" {= "4.02.0-1modular-implicits"} | "camlp4" {= "4.02.0+system"} | "camlp4" {= "4.02.0+2"} | "camlp4" {= "4.02.0+1"} | "camlp4" {= "4.02.0"} | "camlp4" {= "4.01.0"})
```

Comment:
Previously, your package was determined to be uninstallable due to the `camlp4` constraints. 

Now, your package is installable but contains issues later in the install process. In particular, your archive contains more than 1 top level directory. Your archive should contain only a single top level directory.